README.markdown

```diff
@@ -30,7 +30,8 @@ or Wget:
 
     wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh | bash
 
-<sub>The script clones the nvm repository to `~/.nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc` or `~/.profile`).</sub>
+<sub>The script clones the nvm repository to `~/.nvm` and adds the source line to your profile (`~/.bash_profile`, `~/.zshrc` or `~/.profile`).</sub> 
+Eg. `source ~/.profile`
```

Comment:
This instruction is already in the "manual install" section - also it needs to use `.`, not `source`, since that's more portable. What is the benefit of adding this here?